SPECS/kernel/kernel.spec

```diff
@@ -18,6 +18,7 @@ Source3:        sha512hmac-openssl.sh
 Source4:        cbl-mariner-ca-20210127.pem
 Patch0:         0001-clocksource-drivers-hyper-v-Re-enable-VDSO_CLOCKMODE.patch
 Patch1:         0002-add-linux-syscall-license-info.patch
+Patch2:         0003-CVE-2021-43976.patch
```

Comment:
@nisamson will our patch tooling pick up that CVE-2021-43976 has been patched if we have the `0003-` prefix?

Reply:
Is there a preferred naming convention for CVE patches that actually do something? I.e. not a CVE-2012-12345.nopatch?

Reply:
For now, I'll just drop the 000X- naming convention.